amads/algorithms/__init__.py

```diff
@@ -19,7 +19,7 @@
 from ..polyphony.skyline import skyline
 from ..time.durdist1 import duration_distribution_1
 from ..time.durdist2 import duration_distribution_2
-from ..time.meter.break_it_up import ReGrouper
```

Comment:
Perhaps as part of this PR it could be useful to implement an 'example' notebook that illustrates how the functions can be applied for useful analysis approaches?

Reply:
That would be useful ... but perhaps a task for a later PR given David's waiting for this?

Reply:
Yes agreed! Maybe even David provides one as part of his work

Reply:
By all means!